### PR TITLE
Update to blob storage all sdk folders.

### PR DESCRIPTION
**Pull request by sima-zhu**
https://github.com/Azure/azure-sdk-tools/issues/1249

### CODE REVIEW

File: eng/scripts/Language-Settings.ps1 [hunk 1]
```diff
@@ -48,4 +48,7 @@ function Publish-c-GithubIODocs ($DocLocation, $PublicArtifactLocation)
     $pkgInfo = Get-Content $DocLocation/package-info.json | ConvertFrom-Json
     $releaseTag = RetrieveReleaseTag "C" $PublicArtifactLocation
     Upload-Blobs -DocDir $DocLocation -PkgName 'docs' -DocVersion $pkgInfo.version -ReleaseTag $releaseTag
```
Comment (weshaggard) [inline eng/scripts/Language-Settings.ps1:50]: Lets go ahead and remove the docs version.

Comment (sima-zhu) [reply eng/scripts/Language-Settings.ps1]: Done

**Review by weshaggard — APPROVED** [review]

File: eng/scripts/Language-Settings.ps1 [hunk 2]
```diff
@@ -47,5 +47,7 @@ function Publish-c-GithubIODocs ($DocLocation, $PublicArtifactLocation)
     # used to publish multiple docs packages in a single invocation.
     $pkgInfo = Get-Content $DocLocation/package-info.json | ConvertFrom-Json
     $releaseTag = RetrieveReleaseTag "C" $PublicArtifactLocation
-    Upload-Blobs -DocDir $DocLocation -PkgName 'docs' -DocVersion $pkgInfo.version -ReleaseTag $releaseTag
+    Upload-Blobs -DocDir $DocLocation -PkgName 'az_core' -DocVersion $pkgInfo.version -ReleaseTag $releaseTag
+    Upload-Blobs -DocDir $DocLocation -PkgName 'az_iot' -DocVersion $pkgInfo.version -ReleaseTag $releaseTag
+    Upload-Blobs -DocDir $DocLocation -PkgName 'az_storage_blobs' -DocVersion $pkgInfo.version -ReleaseTag $releaseTag
```
Comment (ahsonkhan) [inline eng/scripts/Language-Settings.ps1:52]: @sima-zhu, can you help me understand what is the desired end result of this change?

Comment (weshaggard) [reply eng/scripts/Language-Settings.ps1:52]: The end result is to essentially mimic 3 packages even though there is really only 1. This is align the c repo with other languages when we look at the package index. By duplicating these github.io docs it makes it look like there are 3 packages even though there is really only one.

Comment (sima-zhu) [reply eng/scripts/Language-Settings.ps1:52]: Thanks @weshaggard helping answer this. I missed the comments.

**Pull request merged**